easybuild/easyconfigs/t/tbb/tbb-2021.10.0_GCC_12_fix_compiler_error.patch

```diff
@@ -0,0 +1,24 @@
+# What: Fix compiler error of address XXX will never be NULL.
```

Comment:
We already have this patch: https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/t/tbb/tbb-2021.10.0_null-address.patch